projects/netgrif-components/src/lib/task-content/field-component-resolver/field-component-resolver.component.html

```diff
@@ -1,54 +1,4 @@
 <div fxLayout="row" fxLayoutAlign="start center" class="netgrif-input" [ngClass]="{'min-row-height': isField() && !isCustomHeight()}" [ngSwitch]="getElementType()">
```

Comment:
can't the `ngSwitch` be completely replaced with portals?

Reply:
DataGroup does not have a field implementation and we may want to deprecate the use of them in future resleases. Builder does not supports them neither. A fully portal-based solution would be better, so I decided to replace ngSwitch, and add ngIf for remiaining fields, then if the datagroup solution will be removed, we can remove the div too.